docs/source/tutorials/GEDI/GEDI_L3_tutorial.ipynb

```diff
@@ -0,0 +1,416 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "4e93a6ca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# !pip install geopandas\n",
+    "# !pip install contextily\n",
+    "# !pip install backoff\n",
+    "# !pip install folium\n",
+    "# !pip install geojsoncontour"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "26c4f0f3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from maap.maap import MAAP\n",
+    "import pandas as pd\n",
+    "import folium\n",
+    "from rasterio.plot import show \n",
+    "import rasterio\n",
+    "import boto3\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "95feb677",
+   "metadata": {},
+   "source": [
+    "After importing necessary packages, the next step is to initialize MAAP constructor using `api.ops.maap-project.org` as `maap_host` argument."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "6b322e87",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "maap = MAAP(maap_host=\"api.ops.maap-project.org\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e286167f",
+   "metadata": {},
+   "source": [
+    "**Dataset**\n",
+    "\n",
+    "GEDI L3 Gridded Land Surface Metrics, Version 2\n",
+    "\n",
+    "This dataset provides Global Ecosystem Dynamics Investigation (GEDI) Level 3 (L3) gridded mean canopy height, standard deviation of canopy height, mean ground elevation, standard deviation of ground elevation, and counts of laser footprints per 1-km x 1-km grid cells globally within -52 and 52 degrees latitude.\n",
+    "\n",
+    "Source: https://daac.ornl.gov/cgi-bin/dsviewer.pl?ds_id=1952\n",
+    "\n",
+    "User Guide: https://daac.ornl.gov/GEDI/guides/GEDI_L3_LandSurface_Metrics_V2.html\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c71d10d1",
+   "metadata": {},
+   "source": [
+    "Now, the next step is to seach granules from MAAP CMR. To generate following query, you can use EarthData search feature from MAAP ADE. Refer to this [tutorial](https://docs.maap-project.org/en/latest/tutorials/GEDI/SearchGEDI.html) for more info."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "8a68c929",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# generated from this EDSC search: https://ade.ops.maap-project.org:30052/search?p=!C1201702030-NASA_MAAP&pg[1][v]=t&pg[1][qt]=2015-01-01T00%3A00%3A00.000Z%2C2020-12-31T23%3A59%3A59.999Z&q=L3%20GEDI\n",
+    "results=maap.searchGranule(cmr_host='cmr.maap-project.org',concept_id=\"C1201702030-NASA_MAAP\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ced4d218",
+   "metadata": {},
+   "source": [
+    "The above query gives 1000 results by default. The number of necessary results can be changed using `limit` argument.\n",
```

Comment:
I would just add the limit argument to the search above with the default value. If you think it's important, we can still make a note that the `limit` value is set to the current default